{
    "type": "bundle",
    "id": "bundle--d7a26ebf-dc8f-4e2d-869c-0f6ee2dbebec",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--881ddee0-8f0e-4166-b1ae-32ab0d8b6654",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-03T01:16:22.624277Z",
            "modified": "2025-04-03T01:16:22.624277Z",
            "name": "CVE-2025-21975",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nnet/mlx5: handle errors in mlx5_chains_create_table()\n\nIn mlx5_chains_create_table(), the return value ofÂ mlx5_get_fdb_sub_ns()\nand mlx5_get_flow_namespace() must be checked to prevent NULL pointer\ndereferences. If either function fails, the function should log error\nmessage with mlx5_core_warn() and return error pointer.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-21975"
                }
            ]
        }
    ]
}